app/styles/main.css

```diff
@@ -1,3 +1,19 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer base {
+  :root {
+    --width-hero: 28.75rem;
+  }
+}
+
+h1 {
+  font-size: 2rem;
+  line-height: 2.5rem;
+}
+
+h2 {
+  font-size: 1.125rem;
+  line-height: 1.5rem;
+}
```

Comment:
Seems a bit surprising to me to have Tailwind and also style HTML elements directly via their element selectors. I would have assumed the HTML elements are reset and all styling is happening through class names alone. But I might be misinformed!

Reply:
Not quite. But there is a more tailwind way to define general styling. I changed it!